src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -488,7 +504,9 @@ class FluxPipeline : public DiffusionPipeline {
         m_latent_timestep = timesteps[0];
 
         ov::Tensor latents, processed_image, image_latent, noise;
-        std::tie(latents, processed_image, image_latent, noise) = prepare_latents(initial_image, m_custom_generation_config);
+        float vae_encoder_duration;
+        std::tie(latents, processed_image, image_latent, noise, vae_encoder_duration) = prepare_latents(initial_image, m_custom_generation_config);
+        m_perf_metrics.vae_encoder_inference_duration = vae_encoder_duration;
```

Comment:
can `prepare_latents` write directly to `m_perf_metrics.vae_encoder_inference_duration` ? instead of returning this value from a function and manual assigning later

Reply:
`virtual std::tuple<ov::Tensor, ov::Tensor, ov::Tensor, ov::Tensor> prepare_latents(ov::Tensor initial_image, const ImageGenerationConfig& generation_config) const = 0;`
This function is defined as a const function, so `m_perf_metrics.vae_encoder_inference_duration` can not be written in `prepare_latents` .

Reply:
good point. I think `const` can be removed

Reply:
updated